mgs1600gy_interface/include/mgs1600gy_interface/utils.hpp

```diff
@@ -25,5 +25,8 @@ class Utils
 {
 public:
   static void convertBGR(const std::array<float, 16> &, cv::Mat *, const float, const float);
+  static void convertBRG(
+    const std::array<float, 16> &, std::array<float, 16 * 3> *, const float,
```

Comment:
`std::array<float, 16 * 3> *` -> `std::array<float, 16 * 3>  &` solves the problem.